cql3/statements/describe_statement.cc

```diff
@@ -259,7 +259,14 @@ class keyspace_describe_statement : public describe_statement {
         throw exceptions::invalid_request_exception("No keyspace specified.");
     }
 
-    virtual void postprocess_result(std::unique_ptr<result_set>& result) const override {
+    virtual void postprocess_result(const service::client_state& client_state, std::unique_ptr<result_set>& result) const override {
+        //VALIDATE RESULT
```

Comment:
I think these comments aren't needed, the code is straightforward enough. And if you prefer to keep them, please drop the capslock (:

Reply:
removed